packages/database/src/getConnectionConfig.js

```diff
@@ -3,27 +3,44 @@
  * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
  */
 
-// Note: Must use function to guarantee environment variables have loaded.
-export const getConnectionConfig = () => {
-  return process.env.CI_NAME === 'codeship'
-    ? {
-        host: process.env.CI_TEST_DB_URL,
-        user: process.env.CI_TEST_DB_USER,
-        password: process.env.CI_TEST_DB_PASSWORD,
-        database: process.env.CI_TEST_DB_NAME,
-        ssl: null,
-      }
+const CI_CONNECTION_CONFIG = {
+  host: process.env.CI_TEST_DB_URL,
+  user: process.env.CI_TEST_DB_USER,
+  password: process.env.CI_TEST_DB_PASSWORD,
+  database: process.env.CI_TEST_DB_NAME,
+  ssl: null,
+};
+
+const SERVER_CONNECTION_CONFIG = {
+  host: process.env.DB_URL,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_NAME,
+  ssl: ['true', undefined].includes(process.env.DB_DISABLE_SSL)
```

Comment:
So here is a change: we currently set `ssl` to null only if `DB_DISABLE_SSL` is explicitly set to `true`. It seems to me that it should be the other way around: SSL should be disabled by default, and people can turn it on. But let me know if my understanding is wrong.

If this is correct, we may want to rename this to `DB_ENABLE_SSL` to make this "opt-in" behaviour clearer.

Also, I couldn't find a `.env` file in Last Pass where this was set to a value other than `true`, so I'm wondering whether it is used at all?

Reply:
It's set to false if you are developing locally but want to connect to a remote database. In future, we aim to move the database across to a service like RDS, so we'd also want to use an ssl connection then. 

I'd agree with your suggestion of the rename and default